Makefile

```diff
@@ -663,6 +665,8 @@ ifeq ($(RUN_GPU_TESTS),1)
 		-u 0 \
 		-v $(shell realpath ./run_unit_tests.sh):/ovms/./run_unit_tests.sh \
 		-v $(shell realpath ${GPU_MODEL_PATH}):/ovms/src/test/face_detection_adas/1:ro \
+		-v $(shell realpath ${MNT_LLM_MODELS_PATH}):/mnt/llm_models:ro \
+		-v $(shell realpath ${OVMS_MODELS_PATH}):/opt/home/k8sworker/ovms_models:ro \
```

Comment:
Will this work or fail if there are no such paths on the host? I mean if user runs this will invalid mounts cause failure in entire docker run command?